DataTables.Blazor/DataTable.razor

```diff
@@ -87,11 +87,45 @@
         {
             if (AutoReload && IsDomSourced())
             {
-                await ReInitialiseAsync(); 
+                await ReInitialiseAsync();
             }
         }
     }
 
+    /// <summary>
+    /// Add class to row by index.
+    /// </summary>
+    /// <param name="rowIndex">Index of row.</param>
+    /// <param name="className">Class name.</param>
+    public async ValueTask AddRowClassAsync(int rowIndex, string className)
+        => await Datatables.AddRowClassAsync(_tableReference, rowIndex, className);
```

Comment:
Should be OK to drop the `async` `await` here and just return the `ValueTask` directly which saves allocating a state machine